pkgs/by-name/el/elf-info/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, nix-update-script
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "elf-info";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "kevin-lesenechal";
+    repo = "elf-info";
+    rev = "v${version}";
+    hash = "sha256-wbFVuoarOoxV9FqmuHJ9eZlG4rRqy1rsnuqbGorC2Rk=";
+  };
+
+  cargoHash = "sha256-r4GcJhQn9x5c2hbL+813mS3HbIg8OwNDsMg/fHQoL9Y=";
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "Inspect and dissect an ELF file with pretty formatting";
+    homepage = "https://github.com/kevin-lesenechal/elf-info";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ viperML ];
+    mainProgram = "elf";
+  };
+}
```

Comment:
```suggestion
  meta = {
    description = "Inspect and dissect an ELF file with pretty formatting";
    homepage = "https://github.com/kevin-lesenechal/elf-info";
    license = lib.licenses.gpl3Only;
    maintainers = with lib.maintainers; [ viperML ];
    mainProgram = "elf";
  };
}
```

Reply:
This was the direct output of nix-init, perhaps we should change nix-init to prevent more style nit's.

Reply:
there's literally nothing wrong with with lib;
it's such an ubiquitous attrset that there's no ambiguity where a binding comes from